pkg/webservice/handlers.go

```diff
@@ -98,6 +107,20 @@ func getApplicationsInfo(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func validateQueue(queueName string) error {
+	if queueName != "" {
+		queueNameArr := strings.Split(queueName, ".")
+		for _, name := range queueNameArr {
+			if !configs.QueueNameRegExp.MatchString(name) {
+				return errors.New("Problem in Queue query parameter parsing as queue param '" + queueName + "' " +
+					"contains invalid queue name '" + name + "'. Queue name must only have alphanumeric characters," +
+					" - or _, and be no longer than 64 characters.")
```

Comment:
I would recommend that you generate a new error using `fmt.Errorf()`
In the case that there is just a fixed string you can use `errors.New()` when using a complex construct like you do in this case using `fmt.Errorf()` is far more appropriate and improves readability of the code.